server/src/uuid/model/entity/mod.rs

```diff
@@ -612,6 +614,27 @@ impl Entity {
         .execute(&mut transaction)
         .await?;
 
+        Entity::add_revision(
+            &entity_add_revision_mutation::Payload {
+                input: entity_add_revision_mutation::Input {
+                    changes: payload.input.changes.clone(),
+                    entity_id,
+                    needs_review: false,
+                    subscribe_this: payload.input.subscribe_this,
+                    subscribe_this_by_email: payload.input.subscribe_this_by_email,
+                    fields: payload.input.fields.clone(),
+                },
+                revision_type: EntityRevisionType::from(payload.entity_type.clone()),
+                user_id: payload.user_id,
+            },
+            &mut transaction,
+        )
+        .await?;
+
+        CreateEntityEventPayload::new(entity_id, payload.user_id, instance_id)
+            .save(&mut transaction)
+            .await?;
```

Comment:
Is there a test for this?

Reply:
shouldn't this be before the `Entity::addRevision()`? Otherwise we would have a `AddRevisionEvent` before a `CreateEntityEvent`...

Reply:
good catch
events order fixed in 58297fe
integration test in ed34a2d